starwars/src/List.js

```diff
@@ -0,0 +1,24 @@
+
+import React from 'react';
+
+function Data(props)  {
```

Comment:
Would be a good spot to use object destructuring, since `props` is just an object afterall.

```
function Data({ char }) {
```

Then on line 9, reference `char.map()`